test-runner-frontend/package.json

```diff
@@ -17,6 +17,8 @@
     "buffer": "^6.0.3",
     "firebase": "^9.9.1",
     "firebase-admin": "^11.0.1",
+    "jest-fetch-mock": "^3.0.3",
+    "jspdf-html2canvas": "^1.4.9",
```

Comment:
is this needed anymore?

Reply:
Oh sorry, we no longer need:
`"jest-fetch-mock": "^3.0.3"` neither `"jspdf-html2canvas": "^1.4.9",`